gengokit/httptransport/templates_test.go

```diff
@@ -69,6 +69,9 @@ func EncodeHTTPSumZeroRequest(_ context.Context, r *http.Request, request interf
 	req := request.(*pb.SumRequest)
 	_ = req
 
+	r.Header.Set("transport", "HTTPJSON")
```

Comment:
It's not clear why this is being done in the test, when the exact same thing is being done in the generated client. I'm sure there's a reasonable explanation, but it does not present itself.